ui/src/app/components/buttons.tsx

```diff
@@ -117,13 +117,17 @@ const computeStyle = ({style = {}, hover = {}, disabledStyle = {}}, {disabled})
 
 // Set data test id = '' in the child to prevent it from propegating to all children
 export const Clickable = ({as = 'div', disabled = false, onClick = null, ...props}) => {
+  // Don't propagate test IDs to the rendered child component.
```

Comment:
This change seemed to help with testing – we simply don't propagate the data-test-id prop to any child components, because it will be findable at this component.